src/main/java/rtg/api/config/RTGConfig.java

```diff
@@ -286,6 +283,18 @@
     public RTGConfig() {
 
         //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        // Base Terrain Height
+        //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+        SEA_LEVEL = new ConfigPropertyInt(
+                ConfigProperty.Type.INTEGER,
+                "Sea Level Base Height",
+                "Base Terrain Height",
+                "The 'Y' Level that the World Spawns at.",
+                80, 52, 112
```

Comment:
We can have the default as 80 whilst we're in development, but this will need to be set at 63 when we merge into dev.